### ISSUE TITLE
Not able to finish ubuntu installation

### ISSUE THREAD

**Issue by SDKiller** [ONLYOFFICE/DocumentServer#12]
Ubuntu-1504-vivid-64-minimal

After all packages were installed (installing under root user):

```
...
Setting up onlyoffice-documentserver (3.0.0-99) ...
Warning: Using a password on the command line interface can be insecure.
Warning: Using a password on the command line interface can be insecure.
ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)
ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)
dpkg: error processing package onlyoffice-documentserver (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 onlyoffice-documentserver
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

What exactly configuration script is going to do and where is access denied to **root** user?

Can I fix this manually (set permissions on some folders or so)?

**Comment by SDKiller** [comment]
Seems this is `mysql` error - post-installation script fails to connect to `mysql` server.
How it can be avoided to finish installation?
Mysql server was created during onlyoffice installation.

**Comment by SDKiller** [comment]
Seems this is somehow related to `mysql 5.6` issue: http://skeletor.org.ua/?p=4616

Managed to finish installation manually:

```
 # dpkg  --unpack /var/cache/apt/archives/onlyoffice-documentserver_3.0.0-99_amd64.deb
 # rm /var/lib/dpkg/info/onlyoffice-documentserver.postinst -f
 # apt-get install onlyoffice-documentserver
```

And then - manually creating database from `MySql.CreateDb.sql` and running the rest of `onlyoffice-documentserver.postinst` script.

**Comment by AnaMih** [comment]
Unfortunately, we cannot reproduce the issue. The product was checked using MySql 5.6. Probably, the issue may be caused by some advanced security configuration.

**Comment by trofim24** [comment]
We are release Open Source Version 4.0.0 and fix it.

**Comment by babenkoap** [comment]
I installed OnlyOffice on clean VM behind a proxy, and fixed some issues, which appeared after running post-install script. Before installing onlyoffice-documentserver packet, I made this steps:
1. set up proxy to npm
```
npm config set proxy http://your_proxy:3128
npm config set https-proxy http://your_proxy:3128
```

2. add user onlyoffice to sudo group
`usermod -aG sudo onlyoffice`

You can see, what went wrong after installation, just add `set -x` to /var/lib/dpkg/info/onlyoffice-documentserver.postinst and run manually:
`bash /var/lib/dpkg/info/onlyoffice-documentserver.postinst configure`

**Comment by marcwiesner** [comment]
I had this error hit me this morning. It seemed tied to the mysql password I gave the installer. I had created the db manually (per https://helpcenter.onlyoffice.com/server/linux/document/linux-installation.aspx) and, when the installer asked for the sql password I provided the credentials for the onlyoffice mysql user. 

It wants the root password for mysql. Suggested fix: have the installer prompt the user whether to auto-configure the database (then prompt for root credentials) or connect to a manually configured database (then prompt for non-root credentials and walk away happy).
